path_finding.asciidoc

```diff
@@ -7,11 +7,16 @@ Payment delivery on the Lightning Network depends on finding a path from the sen
 
 The term path finding may be somewhat misleading, because it implies a search for _a single path_ connecting two nodes. In the beginning, when the Lightning Network was small and not well interconnected, the problem was indeed about finding a way to join payment channels to reach the recipient.
 
-But, as the Lightning Network has grown explosively, the path finding problem's nature has shifted. In mid-2021, as we finish this book, the Lightning Network consists of 20,000 nodes connected by at least 55,000 public channels with an aggregate capacity of almost 2,000 BTC. A node has on average 8.8 channels, while the top 10 most connected nodes have between 400 and 2000 channels _each_. A visualizationfootnote:[Produced by explorer.acinq.co from published channel updates] of just a small subset of the Lightning Network channel graph (as of July 2021) is shown in <<lngraph>>:
+But, as the Lightning Network has grown explosively, the path finding problem's nature has shifted. In mid-2021, as we finish this book, the Lightning Network consists of 20,000 nodes connected by at least 55,000 public channels with an aggregate capacity of almost 2,000 BTC. A node has on average 8.8 channels, while the top 10 most connected nodes have between 400 and 2000 channels _each_. A visualization of just a small subset of the Lightning Network channel graph (as of July 2021) is shown in <<lngraph>>:
```

Comment:
In network science the average node degree is usually a pretty meaning less metric as that distribution follows usually a zip'f style power law. I understand that many people might have issues with a power law coefficient thus I suggest that we just depict a histogram of the degree distribution. I am happy to provide one and depict a bit of text (guess the cummulative distribution function will as usual make most sense)

Reply:
also I may add that we could exchange the word explosively to exponentially (though this is note true for some time windows)